community-operators/radanalytics-spark/sparkclusteroperator.0.3.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sparkoperator.v0.3.3
+  namespace: openshift-operators
+  annotations:
+    categories: apache spark, apache, spark
+    certified: "false"
+    description: An operator for managing the Apache Spark clusters and intelligent applications that spawn those clusters.
+    support: jkremser@redhat.com
+    createdAt: 2019-01-17T12:00:00Z
+    containerImage: quay.io/radanalyticsio/spark-operator:latest-released
```

Comment:
I missed this before, but this should be `0.3.3` instead of `latest-released`. Latest released is a "pointer" that currently points to `0.3.3` but during the release process it will automatically point to the newer version and it wouldn't be ideal to have csv file with 0.3.3 in the name using different container image.